lib/ClassNameBuilder.js

```diff
@@ -8,27 +8,24 @@ export default class ClassNameBuilder {
 
     stringify(block, elem, mods, mixes, cls) {
         return this.b_(block, elem, mods) +
-            (mixes? ' ' + mixes.map(mix => mix?
+            (mixes && mixes.length? ' ' + mixes.map(mix => mix?
                 this.b_(mix.block || block, mix.elem, mix.mods) : '').join(' ') :
                 '') + (cls? ' ' + cls : '');
     }
 
-    joinMixes(mix1, mix2) {
```

Comment:
Could you reformulate question? I don't understand it. How many arguments do you expect?